### DISCUSSION TITLE
Feasibility of xpore to work on non-m6A  modification

### DISCUSSION THREAD

**Discussion by YanqiangLi** [GoekeLab/xpore#127] [Q&A] [answered]
Hi, Thanks for your great work for xpore. 
May I ask are there feasibility of xpore to work on non-m6A  modification with WT and knock down/out ? 
Do I need to prepare another model myself? 

Thank you!

**Comment by ploy-np** [comment, marked as answer]
Hi @YanqiangLi, 

Yes, you can use xpore to find the differential modification rates between WT and knock down/out samples for any modification types including non-m6As.

**Reply by YanqiangLi** [reply]
Thanks for your prompt reply. 
Yes, I am working on it. From the first step, it said "Using the signal of unmodified RNA from /programs/x86_64-linux/xpore/2.1/lib/python3.10/site-packages/xpore/diffmod/model_kmer.csv" , from my understanding, here the model_kmer.csv is from the non-m6A modifiled signal, should I prepare the signal of unmodified RNA from IVT to do the analysis when I want to do non-m6A analysis? 
Thank you!
